src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -5269,6 +5269,7 @@ internal static Assembly LoadPSSnapInAssembly(PSSnapInInfo psSnapInInfo)
                 { "Import-Module",                     new SessionStateCmdletEntry("Import-Module", typeof(ImportModuleCommand), helpFile) },
                 { "Invoke-Command",                    new SessionStateCmdletEntry("Invoke-Command", typeof(InvokeCommandCommand), helpFile) },
                 { "Invoke-History",                    new SessionStateCmdletEntry("Invoke-History", typeof(InvokeHistoryCommand), helpFile) },
+                { "Measure-Script",                    new SessionStateCmdletEntry("Measure-Script", typeof(MeasureScriptCommand), helpFile) },
```

Comment:
`Trace-Script` seems like a better name for what it does currently.  A separate `Measure-Script` may be needed later that leverages `Trace-Script`, but produces summarized output to the user.

Reply:
I agree. 

`Measure-Script` is the name of the main cmdlet in PSProfiler. `Trace-Script` is the name of the main cmdlet in my Profiler module, **but** only because Measure-Script is already taken. Both these cmdlets provide a summary.

So using `Trace-Script` here to get the "raw" data, is imho the best choice. And I am okay with renaming mine.